terraform/team-members-datadog/_terraform.tf

```diff
@@ -0,0 +1,22 @@
+// Configuration for Terraform itself.
+
+terraform {
+  required_version = "~> 1"
+
+  required_providers {
+    datadog = {
+      source  = "datadog/datadog"
+      version = "3.24.1"
```

Comment:
And perhaps pinning this version to include all the patch ones: `~> 3.24`

Reply:
There's a broader discussion coming up once in a while about our infrastructure-as-code tooling and its issues and limitations. One of the things that we're struggling with is keeping our modules and dependencies up-to-date. It's not too bad here, but for other modules it's quite a lot of work to update something, because then every service that depends on that module needs to be manually redeployed.

I think it's fine to leave it as-is for now, since we really need to solve this in an holistic way for the whole repository.